app/models/edm/web_resource.rb

```diff
@@ -140,6 +147,13 @@ def europeana_supported_media_mime_type
       end
     end
 
+    def media_size
```

Comment:
Could we rename this to something like `media_size_permitted`? (I would expect `EDM::WebResource#media_size` to return an instance's media size.)